config/lint/.markdown-lint.yaml

```diff
@@ -0,0 +1,77 @@
+# Start with default values
+# Ref: https://github.com/DavidAnson/markdownlint/blob/main/schema/.markdownlint.yaml
+default: true
+
+# MD003/heading-style : Heading style : https://github.com/DavidAnson/markdownlint/blob/main/doc/md003.md
+MD003:
+  # Heading style
+  style: "atx"
+
+# MD007/ul-indent : Unordered list indentation
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md007.md
+MD007:
+  indent: 4
+
+# MD013/line-length : Line length
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md
+MD013:
+  # Number of characters
+  line_length: 80
+  # Do not apply line length rule to code blocks, tables or headings
+  # Links and images are excluded by default.
+  code_blocks: false
+  tables: false
+  headings: false
+  # Don't enable strict or stern modes because we might have long URLs
+  strict: false
+  stern: false
+
+# MD020/no-missing-space-closed-atx
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md021.md
+# We don't use atx-style headings, so no need to enforce this rule
+MD020: false
+
+# MD021/no-multiple-space-closed-atx
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md021.md
+# We don't use atx-style headings, so no need to enforce this rule
+MD021: false
+
+# MD029/ol-prefixt : Ordered list item prefix
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md029.md
+MD029:
+  style: one
+
+# MD030/list-marker-space : Spaces after list markers
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md030.md
+# Let prettier handle indentation until
+# https://github.com/prettier/prettier/issues/16929 is solved
+MD030: false
```

Comment:
The rules that are commented below look like they are equivalent to the old rules.

Reply:
Correct. Once the referenced issue is solved, we can enable it as it was before.